data/json/proficiencies/tailoring.json

```diff
@@ -28,7 +28,8 @@
     "can_learn": true,
     "default_time_multiplier": 2,
     "default_fail_multiplier": 1.25,
-    "time_to_learn": "10 h"
+    "time_to_learn": "10 h",
+    "required_proficiencies": [ "prof_closures" ]
```

Comment:
This is intentionally left off. The prof initially required closures but as time went on it became clear we needed a "waterproofing" prof unrelated to closures, and there wasn't room for two similar proficiencies like that.